src/Robot-Dobot_MG400/main.py

```diff
@@ -125,6 +125,11 @@ def set_GUIparameter(self):
                         "Payload x offset": 0.0,
                         "Payload y offset": 0.0,
                         # "Payload z offset": 0.0,
+                        " ": None,
+                        "Jump": None,
+                        "Use jump mode": False,
+                        "Movement height": 0.0,
```

Comment:
maybe explain new parameters in description

Also there is a class variable "axes" which seems to be used nowhere?

Reply:
addded further description and comment